docs/scos/dev/upgradability-services/upgradability-reference.md

```diff
@@ -110,3 +110,201 @@ The method name should be unique to the extent of making it impossible to accide
 2. Override the core method with the custom one you've created in the previous step.
 
 When the core method is overriden with a custom one, re-evaluate the code. The same error shouldn't be returned.
+
+
+
+## Non-public API class was extended or used
+
+Private API updates can break backward compatibility. So, backward compatibility in minor releases is not guaranteed in the private API. For example, if you use a core class or method on the project level, and it is updated or removed, the error may occur during an update.
```

Comment:
`...the error may occur during an update.` Could you please explain what exactly can break? For example, the extended class might change behavior.